examples/monaco-editor-example/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "create-foxglove-extension/tsconfig/tsconfig.json",
```

Comment:
Is this extends right? I thought we use foxglove/tsconfig?

Reply:
That's what you get out of the box from running:

```bash
npm init foxglove-extension@latest helloworld
```

Do we need to change it?

Reply:
Or maybe the idea was to provide a tsconfig specific to extensions authoring. I could see that having been the thinking for this.

Reply:
Yeah I can imagine that might be useful.